src/lib/postcssPlugin.ts

```diff
@@ -27,11 +27,13 @@ function collectCustomSyntaxFrom(value: string, customSyntaxStrs): void {
 
 export const extractTPACustomSyntax = postcss.plugin('postcss-wix-tpa-style', (opts: IOptions = {} as IOptions) => {
   const cssVars = {};
-  const customSyntaxStrs = [];
+  const customSyntaxStrs: string[] = [];
 
   return (css: ContainerBase) => {
     css.walkDecls((decl: Declaration) => {
-      Object.keys(replacers).forEach(replacerName => (decl = replacers[replacerName](decl)));
+      const replacersNames = (Object.keys(replacers) as unknown) as (keyof typeof replacers)[];
```

Comment:
this is just a cast for `Object.keys()` (it losses types).

Could use `Object.entries()` (that will propagate types) but for some versions of Node that could be unsupported.